templates/registration/login.html

```diff
@@ -1,12 +1,22 @@
 {% extends 'base.html' %}
-{% block title %}Login{% endblock %}
+
+{% block title %}Imaging experiments tracking | Login{% endblock %}
```

Comment:
Why would we prepend titles with Imaging experiments tracking? This way it gets harder to identify which tab is that in the browser without clicking on it. To identify a website, it's better to add a site-wide favicon. Could you do it? I think just an image of a letter "I" or "Im" will go for now.

Reply:
Adding the name of the page to the titles is a SEO thing that we don't need. I'll add a favicon.